client/src/main.rs

```diff
@@ -27,6 +28,10 @@ fn main() {
         let cert = cert.serialize_der().unwrap();
         let sim_cfg = config.local_simulation.clone();
 
+        let save = dirs.data_local_dir().join("default.save");
+        info!("using save file {}", save.display());
+        let save = Save::open(&save, config.local_simulation.chunk_size).unwrap();
```

Comment:
On my first attempt running this without doing any prior setup, I got the following error:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Db(DbError(Io(Os { code: 3, kind: NotFound, message: "The system cannot find the path specified." })))', client\src\main.rs:33:74
```

If I manually create the "data" directory, this error goes away. As far as I can tell, `redb` doesn't have a way to ensure that the directory exists when creating the directory, so to avoid needing additional setup, an mkdirs-equivalent call is likely needed.

Reply:
Good catch; added a call to `create_dir_all`.